docs/ipfs-protocol-handlers.md

```diff
@@ -0,0 +1,300 @@
+# ✨🪐 IPFS Fetch API 🪐✨
+
+The `fetch()` API browsers use for HTTP, but for IPFS!
+**[Intro Video📺](https://youtu.be/kI9Issf3MNc?t=1629)**
+
+### Example
```

Comment:
Might be good to have a not here on how to open devtools and paste the example in to get a result